[2023-01-26 02:12:41,690] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: d_1_finalproject.get_data_from_api manual__2023-01-26T02:12:40.488687+00:00 [queued]>
[2023-01-26 02:12:41,697] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: d_1_finalproject.get_data_from_api manual__2023-01-26T02:12:40.488687+00:00 [queued]>
[2023-01-26 02:12:41,698] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 02:12:41,699] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-26 02:12:41,699] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 02:12:41,708] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_data_from_api> on 2023-01-26 02:12:40.488687+00:00
[2023-01-26 02:12:41,713] {standard_task_runner.py:52} INFO - Started process 3701 to run task
[2023-01-26 02:12:41,718] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'd_1_finalproject', 'get_data_from_api', 'manual__2023-01-26T02:12:40.488687+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/d_1_finalproject_dag.py', '--cfg-path', '/tmp/tmp556ox87p', '--error-file', '/tmp/tmpdmcu89hr']
[2023-01-26 02:12:41,719] {standard_task_runner.py:80} INFO - Job 36: Subtask get_data_from_api
[2023-01-26 02:12:41,771] {task_command.py:369} INFO - Running <TaskInstance: d_1_finalproject.get_data_from_api manual__2023-01-26T02:12:40.488687+00:00 [running]> on host 7580358f394d
[2023-01-26 02:12:41,829] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=d_1_finalproject
AIRFLOW_CTX_TASK_ID=get_data_from_api
AIRFLOW_CTX_EXECUTION_DATE=2023-01-26T02:12:40.488687+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-26T02:12:40.488687+00:00
[2023-01-26 02:12:42,020] {covid_scraper.py:12} INFO - GET DATA FROM API COMPLETED
[2023-01-26 02:12:42,027] {covid_scraper.py:14} INFO - DATA FROM API TO DATAFRAME READY
[2023-01-26 02:12:42,039] {logging_mixin.py:115} INFO - <class 'pandas.core.frame.DataFrame'>
RangeIndex: 145 entries, 0 to 144
Data columns (total 20 columns):
 #   Column                    Non-Null Count  Dtype 
---  ------                    --------------  ----- 
 0   CLOSECONTACT              145 non-null    int64 
 1   CONFIRMATION              145 non-null    int64 
 2   PROBABLE                  145 non-null    int64 
 3   SUSPECT                   145 non-null    int64 
 4   closecontact_dikarantina  145 non-null    int64 
 5   closecontact_discarded    145 non-null    int64 
 6   closecontact_meninggal    145 non-null    int64 
 7   confirmation_meninggal    145 non-null    int64 
 8   confirmation_sembuh       145 non-null    int64 
 9   kode_kab                  145 non-null    object
 10  kode_prov                 145 non-null    object
 11  nama_kab                  145 non-null    object
 12  nama_prov                 145 non-null    object
 13  probable_diisolasi        145 non-null    int64 
 14  probable_discarded        145 non-null    int64 
 15  probable_meninggal        145 non-null    int64 
 16  suspect_diisolasi         145 non-null    int64 
 17  suspect_discarded         145 non-null    int64 
 18  suspect_meninggal         145 non-null    int64 
 19  tanggal                   145 non-null    object
dtypes: int64(15), object(5)
memory usage: 22.8+ KB
[2023-01-26 02:12:42,040] {logging_mixin.py:115} INFO - None
[2023-01-26 02:12:42,131] {authentication.py:70} INFO - package: ***.connector.plugins
[2023-01-26 02:12:42,133] {authentication.py:71} INFO - plugin_name: caching_sha2_password
[2023-01-26 02:12:42,135] {authentication.py:75} INFO - AUTHENTICATION_PLUGIN_CLASS: MySQLCachingSHA2PasswordAuthPlugin
[2023-01-26 02:12:42,288] {logging_mixin.py:115} INFO - DATA INSERTED SUCCESSFULLY TO MYSQL
[2023-01-26 02:12:42,289] {python.py:173} INFO - Done. Returned value was: None
[2023-01-26 02:12:42,319] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=d_1_finalproject, task_id=get_data_from_api, execution_date=20230126T021240, start_date=20230126T021241, end_date=20230126T021242
[2023-01-26 02:12:42,380] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-26 02:12:42,414] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
